controller/src/beerocks/master/wfa_ca.cpp

```diff
@@ -280,7 +282,42 @@ void wfa_ca::handle_wfa_ca_message(
         break;
     }
     case eWfaCaCommand::DEV_GET_PARAMETER: {
-        // TODO
+
+        /* 
+        * This command retrieves the requested parameter information and sends the information in the response.
+        *
+        * Parameters:
+        * 
+        * Param Name    | Values                        | Description
+        * ----------------------------------------------------------------------------------------
+        * "program"     | string (MAP)                  | Program name
+        * "ruid"        | string                        | A radio Unique ID
+        * "SSID"        | string                        | The SSID that fronthaul AP is configured. When SSID is not specified for the parameter
+                                                          "macaddr", it implies that device should return backhaul STA's MAC address. SSID is also
+                                                          not required for the parameter "ALid"
+        * 
+        * Return Values: 
+        *   None.
+        *
+        * Example:
+        *  dev_get_parameter,program,map,ruid,WTS_REPLACE_RUID,ssid,Multi-AP-24GC-1,parameter,macaddr
+        *  status,COMPLETE,macaddr,11:22:33:44:55:66
+        *  dev_get_parameter,program,map,ruid,WTS_REPLACE_RUID,ssid,Multi-AP-24GC-1,parameter,bssid
+        *  status,COMPLETE,bssid,11:22:33:44:55:66
+        */
+
+        // For controller certification, the following command is received to get the ALid:
+        // DUT (127.0.0.1:5000) ---> dev_get_parameter,program,map,parameter,ALid
+
+        if (!reply(sd, cmdu_tx, eWfaCaStatus::RUNNING)) {
+            LOG(ERROR) << "failed to send reply";
+            break;
+        }
+    
+        std::string alid;
+        if (!net::network_utils::linux_iface_get_mac("br-lan", alid))
+            reply(sd, cmdu_tx, eWfaCaStatus::INVALID, "FAIL");
```

Comment:
Since this failure is not related to command validity and occurs after you send status RUNNING, I should be ERROR instead of INVALID.
Also please add debug log, and break.